### PR TITLE
Adds an additional effect to green slime cores

### PR DESCRIPTION
**Pull request by ghost**
This PR adds unstable mutation toxin. By injecting a green slime extract with radium, it will create this new toxin. It is similar to the normal mutation toxin in that it mutates those injected with it into a different race. However, unlike the stable toxin, this is completely unpredictable. It will change their race to a random (cosmetic) race, from lizardman to flyman. There is a 10% chance that instead they will remain a human. This works on all races except for golems.

The races included in the whitelist only have minor effects on gameplay.A person's game will not be inherently changed by imbibing this toxin, and in most circumstances it will be voluntary anyway.

### CODE REVIEW

**Comment by PKPenguin321** [comment]
why not let it be used by all races? it could be the rare cure for being made into a flyman by the teleporter, or grant a golem it's humanity

**Comment by Xhuis** [comment]
Mainly so people can't keep jabbing themselves with it over and over to get the race they want. As for golems, it wouldn't make any sense. It's also the same way with normal mutation toxin so I kept it in that regard.

**Comment by Man-Shroom** [comment]
>Only usable on humans

BOOOO HOW CAN I FORCE LIGGERS TO BECOME HUMAN NOW

**Comment by Incoming5643** [comment]
I don't see how letting people continuously reroll with it would be a problem, even from a full list of races. None of them are exceptionally powerful. Shadowperson is pretty much the worst race in the world to be because you can't even exist in a workplace unless you have Night vision goggles and break the lights. Getting stuck like that is a terrible fate.

File: code/modules/mob/living/carbon/human/species_types.dm [hunk 1]
```diff
@@ -17,7 +17,24 @@
...
+		sleep(30)
+		var/list/blacklisted_species = list(/datum/species/zombie, /datum/species/skeleton, /datum/species/human, /datum/species/golem, /datum/species/golem/adamantine)
+		var/list/possible_morphs = typesof(/datum/species/) - blacklisted_species
+		var/datum/species/mutation = pick(possible_morphs)
+		if(prob(90) && mutation)
+			H << "<span class='danger'>The pain subsides, and you slowly realize you are no longer human...</span>"
+			H.dna.species = new mutation()
+			H.regenerate_icons()
+			if(mutation == /datum/species/slime)
+				H.faction |= "slime"
```
Comment (Incoming5643) [inline code/modules/mob/living/carbon/human/species_types.dm]: I'm pretty sure you don't need to do this, the simple act of adding the species to the person does it.

Comment (Xhuis) [inline code/modules/mob/living/carbon/human/species_types.dm]: I don't think so, the normal green slime extract does the same thing. I did it here just to be safe.

Also, I'm probably gonna blacklist shadowperson from this list.

**Comment by ACCount12** [comment]
I think it must be for all races, not just for humans.

**Comment by Alek2ander** [comment]
How about if you inject it again too soon, you don't transform but get irradiated instead?

**Comment by AlexCrimson** [comment]
I think limiting it to humans is best. Letting people endlessly reroll for race seems gamey. Even if the race stuff is just cosmetic.

**Comment by Kromgar** [comment]
I think you should be able to inject this into slimes to turn the human. It would be great to undo forced slimings

**Comment by Incoming5643** [comment]
I have another chem I'm working on for that, basically the water reaction for black slimes would turn full slimes into slimepeople, and the water reaction for green slimes would turn mutant races (back) into humans.

**Comment by Xhuis** [comment]
All right, shadow people have been blacklisted from the possible race mutations. I'm not going to allow other races to stick themselves with this stuff over and over to keep changing until they can be a plant because it not only takes up a lot of time but also a lot of slime cores.

**Comment by Kromgar** [comment]
Just make it only lizards and humans can use it

**Comment by Xhuis** [comment]
If people _really_ want to keep jabbing themselves with it, they can have someone debrain them, put 'em into a human body, and clone that body. Fluff-wise, the toxin only reacts with human DNA before becoming inert.

**Comment by Kromgar** [comment]
But your ruining the emergent gameplay of being a dickass ligger hater who turns them into anything but a lizard. If it allows lizards to prick themselves.

Well atleast autocloner revives you as human if you suicide as a lizard

**Comment by optimumtact** [comment]
This is just gonna be a grief tool for turning other people into unwanted races.

**Comment by Incoming5643** [comment]
So's the other reaction for green slime cores. Golden rule of the game right there: "if you let someone inject something into you, you may be good as dead"

**Comment by Xhuis** [comment]
Made the toxin work on all species

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents.dm [hunk 2]
```diff
@@ -368,6 +368,34 @@ datum/reagent/slimetoxin
...
+	description = "An unstable and unpredictable corruptive toxin produced by slimes."
+	color = "#5EFF3B" //RGB: 94, 255, 59
+	metabolization_rate = INFINITY //So it instantly removes all of itself
+
+datum/reagent/unstableslimetoxin/on_mob_life(var/mob/living/carbon/human/H as mob)
+	..()
+	H << "<span class='warning'><b>You crumple in agony as your flesh wildly morphs into new forms!</b></span>"
+	H.visible_message("<b>[H]</b> falls to the ground and screams as their skin bubbles and froths!") //'froths' sounds painful when used with SKIN.
+	H.Weaken(3)
+	sleep(30)
```
Comment (phil235) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents.dm]: WHYYYYYYYYYYYYYYYYYYYY

Comment (optimumtact) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents.dm]: who are you asking? The original pr maker is gone

Comment (phil235) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents.dm]: Noone, I'm pointing the finger to vent my frustration from the shit code I have to suffer through.

Comment (optimumtact) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents.dm]: I h :ear: you

**Pull request merged**